test/metabase/util/queue_test.clj

```diff
@@ -35,8 +38,8 @@
       (try
         (while true
           ;; Stop the consumer once we are sure that there are no more events coming.
-          (u/with-timeout 100
```

Comment:
It was weird for this timeout to be shorter than the one we use with `blocking-take!` below. This low value was probably the cause.